akka-docs/src/main/paradox/stream/stages-overview.md

```diff
@@ -730,6 +730,31 @@ Transform each element in the stream by calling a mapping function with it and p
 
 **completes** when upstream completes
 
+---------------------------------------------------------------
+
+### inspect
+
+Similar to `map`, however does not modify the passed through element, the returned value is ignored.
+
+This operation is useful for performing inspecting the passed through element, usually by means of side-effecting operations (such as `println`, or emitting metrics), for each element without having to modify it.
```

Comment:
performing inspecting sounds funny, some other wording?, or simply s/performing inspecting/inspecting/

Reply:
Reading these again it still "calls for" simply being called `foreach`.. and simply `performing side effects` wording here..